features/cs3api-validator.feature

```diff
@@ -0,0 +1,9 @@
+Feature: list home space
+  To store files
+  As a consumer of the cs3api
+  I need to be able obtain my personal space
+
+  Scenario: Simple Login
+    Given a personal space was created for user admin
+    When I call listStorage Spaces
+    Then there should be one personal space in the response
```

Comment:
I would suggest to stick to the battle proved way of writing the feature files in here
https://doc.owncloud.com/server/10.9/developer_manual/testing/acceptance-tests.html#feature-files
that would make consistent with the existing tests and reduce needed brain power when switching over

```suggestion
Feature: list home space
  As a consumer of the cs3api
  I want to obtain my personal space
  So that I can store files

  Scenario: List space storage
    Given user "admin" has created a personal space
    When user "admin" lists all available spaces
    Then one personal space should be listed in the response
```